.buildkite/scripts/backport_branch.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+
+source .buildkite/scripts/common.sh
+
+set -euo pipefail
+
+if buildkite-agent meta-data exists DRY_RUN; then
+  DRY_RUN="$(buildkite-agent meta-data get DRY_RUN)"
+else
+  DRY_RUN=${DRY_RUN:-"true"}
+fi
```

Comment:
Nit. I wonder if we can create some kind of helper for this code that we use to get variables.
```suggestion
DRY_RUN=$(get_buildkite_metadata DRY_RUN "$DRY_RUN" "true")
```

With an implementation like this one:
```
function get_buildkite_metadata() {
  local var_name=$1
  local current="$2"
  local default="$3"

  if buildkite-agent meta-data exists $var_name; then
    echo "$(buildkite-agent meta-data get $var_name)"
  else
    echo ${current:-$default}
  fi
}
```

Reply:
Or, it seems to be possible to pass default values to this command.
```suggestion
DRY_RUN="$(buildkite-agent meta-data get DRY_RUN --default ${DRY_RUN:-"true"})"
```
https://buildkite.com/docs/agent/v3/cli-meta-data#getting-data

Reply:
Thanks @jsoriano, great point!
fixed!